CONTRIBUTING.md

```diff
@@ -271,6 +271,8 @@ To validate the documentation files, run:
 make lintdoc
 ```
 
+Note: doxygen >= 1.9.5 is required.
```

Comment:
Would it be too lazy to say something like "The minimum version is documented at the top scripts/gen_vimdoc.py" or something? We don't really document the minimum uncrustify version and such either. Just an idea, feels like it'd require less maintenance.

Reply:
I agree! We could just drop this line and minimize maintenance points, because anyway users will get error messages when running `make doc`.